### PR TITLE
Add Windows dynamic libraries

### PR DESCRIPTION
**Pull request by shadowwalkersb**

### CODE REVIEW

**Comment by shadowwalkersb** [comment]
CircleCI log:

```
Adding distribution /feedstock_root/build_artefacts/linux-64/libssh2-1.7.0-0.tar.bz2 to conda-forge's main channel
Traceback (most recent call last):
  File "/opt/conda/bin/upload_or_check_non_existence", line 140, in <module>
    main()
  File "/opt/conda/bin/upload_or_check_non_existence", line 129, in main
    add_distribution_to_channel(cli, meta, owner, channel)
  File "/opt/conda/bin/upload_or_check_non_existence", line 98, in add_distribution_to_channel
    binstar_cli.add_channel(channel, owner, meta.name(), meta.version())
  File "/opt/conda/lib/python3.5/site-packages/binstar_client/mixins/channels.py", line 47, in add_channel
    self._check_response(res, [201])
  File "/opt/conda/lib/python3.5/site-packages/binstar_client/__init__.py", line 200, in _check_response
    raise ErrCls(msg, res.status_code)
binstar_client.errors.NotFound: ('you do not have write permissions to package conda-forge/libssh2', 404)

./ci_support/run_docker_build.sh returned exit code 1
```

**Comment by jakirkham** [comment]
Looks like you enabled CircleCI on your fork. Will need to disable that. Afterwards try pushing some sort of new commit here.

**Comment by shadowwalkersb** [comment]
Will do that. But, why does CircleCI need to be disabled?

**Comment by jakirkham** [comment]
Because it is getting confused and running on your fork instead of here at conda-forge. As a result we get these permission complaints.

**Comment by shadowwalkersb** [comment]
Another question, why is it disabled on conda-forge?

**Comment by shadowwalkersb** [comment]
> Looks like you enabled CircleCI on your fork. Will need to disable that.

Does this mean, disable this project on CircleCI, in other words, unfollow project? Nothing to do with disabling linux builds.

**Comment by jakirkham** [comment]
> Another question, why is it disabled on conda-forge?

It was hidden by your fork having the project enabled. Now that you have disabled CircleCI on your fork it works fine at conda-forge.

> Does this mean, disable this project on CircleCI, in other words, unfollow project? Nothing to do with disabling linux builds.

I think you have figured this out given it is working again.

**Comment by shadowwalkersb** [comment]
Is this a problem, https://ci.appveyor.com/project/conda-forge/libssh2-feedstock/build/1.0.4/job/xcra9t5bevmcj3d8#L119 - https://ci.appveyor.com/project/conda-forge/libssh2-feedstock/build/1.0.4/job/xcra9t5bevmcj3d8#L181. It's happening on Appveyor a lot recently.

**Comment by shadowwalkersb** [comment]
ping

**Comment by jakirkham** [comment]
Unrelated FYI have enabled CircleCI to build PRs on this repo. So it should no longer be necessary to have CircleCI enabled on your fork.

**Comment by jakirkham** [comment]
> Is this a problem, https://ci.appveyor.com/project/conda-forge/libssh2-feedstock/build/1.0.4/job/xcra9t5bevmcj3d8#L119 - https://ci.appveyor.com/project/conda-forge/libssh2-feedstock/build/1.0.4/job/xcra9t5bevmcj3d8#L181. It's happening on Appveyor a lot recently.

Ugh thought we had fixed that. ( https://github.com/conda-forge/conda-smithy/issues/351 ) 😒

Seems related to `conda` 4.1, but went away with `conda` 4.2. The downgrade is likely happening because of our old copy of `ruamel_yaml`.

**Comment by jakirkham** [comment]
> Is this a problem, https://ci.appveyor.com/project/conda-forge/libssh2-feedstock/build/1.0.4/job/xcra9t5bevmcj3d8#L119 - https://ci.appveyor.com/project/conda-forge/libssh2-feedstock/build/1.0.4/job/xcra9t5bevmcj3d8#L181. It's happening on Appveyor a lot recently.

Wait this build is 3 months old. 😕 I'm not seeing this in any recent builds. Are you?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -24,6 +24,7 @@ requirements:
     - cmake >2.8
     - openssl 1.0.*
     - toolchain
+    - vs2008_express_vc_python_patch  # [win64]
```
Comment (jakirkham) [inline recipe/meta.yaml:27]: This is now applied in `conda-forge-build-setup`. So no need to depend on it or use it here.

**Pull request closed without merging**